admin/tabs/activity/clienterrors/ClientErrorsModel.js

```diff
@@ -216,31 +215,29 @@ export class ClientErrorsModel {
 
     @action
     adjustDates(dir) {
-        const tomorrow = LocalDate.tomorrow(),
-            start = this.startDate,
-            end = this.endDate,
+        const appDay = LocalDate.currentAppDay(),
+            start = this.startDay,
+            end = this.endDay,
             diff = end.diff(start),
             incr = diff + 1;
 
         let newStart = start[dir](incr),
             newEnd = end[dir](incr);
 
-        if (newEnd > tomorrow) {
```

Comment:
"tomorrow" was being used to work around the timezone issue? Is that right? Now everyone gets the same app days so we don't need the buffer?

Reply:
NVM I see the comment below.